### ISSUE TITLE
FilePresenter should be removed when app enters background

### ISSUE THREAD

**Issue by pronebird** [drewmccormack/ensembles#231]
Hi,

There is an interesting note in iOS documentation regarding File Presenters on iOS:

https://developer.apple.com/reference/foundation/nsfilecoordinator?language=objc

Excerpt: 

>File Presenters and iOS
If your app enters the background with an active file presenter, any other processes that perform a coordinated read or write on the presented file can deadlock. To prevent this situation, call removeFilePresenter: to remove the file presenter in the applicationDidEnterBackground: method or in response to a UIApplicationDidEnterBackgroundNotification notification. Call addFilePresenter: to add the file presenter again in the applicationWillEnterForeground: method or in response to a UIApplicationWillEnterForegroundNotification notification.

Looking at `CDEICloudFileSystem` I don't see it actually removing or adding itself in response to any of the notifications. Is it some legacy notice or is this information is still relevant to latest iOS?

**Comment by drewmccormack** [comment]
Indeed. Looks like that is something we should change. Will look into it.
